src/mono/mono/mini/interp/transform.c

```diff
@@ -4554,7 +4587,8 @@ generate_code (TransformData *td, MonoMethod *method, MonoMethodHeader *header,
 		exit_bb->stack_height = -1;
 	}
 
-	get_basic_blocks (td, header, td->gen_sdb_seq_points);
+	il_targets = mono_bitset_mem_new(mono_mempool_alloc0(td->mempool, mono_bitset_alloc_size(header->code_size, 0)), header->code_size, 0);
+	get_basic_blocks (td, header, td->gen_sdb_seq_points, il_targets, error);
```

Comment:
we should check error status here

Reply:
if `get_basic_blocks` return FALSE, `td->has_invalid_code` is set to TRUE and execution is terminated